ixmp/core.py

```diff
@@ -164,6 +164,48 @@ def scenario_list(self, default=True, model=None, scen=None):
         return pd.DataFrame(self._backend.get_scenarios(default, model, scen),
                             columns=FIELDS['get_scenarios'])
 
+    def export_timeseries_data(self, path, default=True, model=None,
+                               scenario=None, variables=None):
+        """Bulk timeseries export to CSV file.
```

Comment:
- I would avoid introducing the word "bulk", which we don't define and don't use anywhere else.
- Explain that only data added with `add_timeseries` (link to the method) is exported.